eclipse-test/swtbot-eclipse-test/pom.xml

```diff
@@ -0,0 +1,117 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd"
+	xmlns="http://maven.apache.org/POM/4.0.0"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+	<modelVersion>4.0.0</modelVersion>
+	<artifactId>devonfw-swtbot-eclipse-test</artifactId>
+	<name>Eclipse Plug-In Tests</name>
+	<packaging>eclipse-test-plugin</packaging>
+	<version>3.2.1</version>
+
+	<parent>
+		<groupId>com.devonfw.tools.ide</groupId>
+		<artifactId>devonfw-eclipse-test</artifactId>
+		<version>dev-SNAPSHOT</version>
+	</parent>
+
+	<properties>
+		<tycho.testArgLine>-Xms512m -Xmx2048m -XX:+DisableExplicitGC</tycho.testArgLine>
+	</properties>
+
+	<dependencies>
+
+	</dependencies>
+
+	<build>
+		<plugins>
+			<!-- Maven Build -->
+			<plugin>
+				<groupId>org.apache.maven.plugins</groupId>
+				<artifactId>maven-dependency-plugin</artifactId>
+				<executions>
+					<execution>
+						<id>copy-dependencies</id>
+						<phase>initialize</phase>
+						<goals>
+							<goal>copy-dependencies</goal>
+						</goals>
+						<configuration>
+							<includeScope>runtime</includeScope>
+							<overWriteIfNewer>true</overWriteIfNewer>
+							<outputDirectory>${project.basedir}/lib</outputDirectory>
+							<markersDirectory>${project.basedir}/lib</markersDirectory>
+						</configuration>
+					</execution>
+				</executions>
+			</plugin>
+			<plugin>
+				<groupId>org.apache.maven.plugins</groupId>
+				<artifactId>maven-clean-plugin</artifactId>
+				<version>2.5</version>
+				<executions>
+					<execution>
+						<id>clean-dependencies</id>
+						<phase>clean</phase>
+						<goals>
+							<goal>clean</goal>
+						</goals>
+						<configuration>
+							<filesets>
+								<fileset>
+									<directory>lib</directory>
+									<includes>
+										<include>**/*</include>
+									</includes>
+								</fileset>
+							</filesets>
+						</configuration>
+					</execution>
+				</executions>
+			</plugin>
+			<!-- Tycho Build -->
+			<plugin>
+				<groupId>org.eclipse.tycho</groupId>
+				<artifactId>target-platform-configuration</artifactId>
+				<configuration>
+					<dependency-resolution>
+						<extraRequirements>
+							<requirement>
+								<type>p2-installable-unit</type>
+								<id>org.eclipse.m2e.feature.feature.group</id>
+								<versionRange>0.0.0</versionRange>
+							</requirement>
+
+						</extraRequirements>
+					</dependency-resolution>
+				</configuration>
+			</plugin>
+			<plugin>
+				<groupId>org.eclipse.tycho</groupId>
+				<artifactId>tycho-surefire-plugin</artifactId>
+				<version>${tycho.version}</version>
+				<configuration>
```

Comment:
This test just executes tests on an already available available project, but it does not test the eclipse which has been created by the devonfw-ide at all. Are you aware of that? Is that intended?
This configuration says, please download the latest eclipse sdk release based on the repository you gave below (photon).

I think you want to test the devonfw-ide created eclipse instance right? Then please have a look at https://wiki.eclipse.org/Tycho/Testing_with_Surefire#p2Installed_on_provisioned_RCP_application

Reply:
Have you already seen the integration tests of `devonfw-ide`? Maybe you should integrate with that to avoid reinventing the wheel...

Reply:
Hi @maybeec , Should i use devon-ide eclipse local instance which will be already installed by developer?